zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/archiver/OpenSearchRepositoryIT.java

```diff
@@ -151,6 +240,8 @@ private RestClientTransport createRestClient() {
     return new RestClientTransport(restClient, new JacksonJsonpMapper());
   }
 
+  private record TestBatchOperation(String id, String endDate) implements TDocument {}
```

Comment:
:x: I guess this is a left over from another PR?

Reply:
Solved with the rabase as well